### PR TITLE
Cleanup of SocketManager (struct -> namespace).

### PR DESCRIPTION
**Pull request by ywkaras**

### CODE REVIEW

File: iocore/eventsystem/I_SocketManager.h [hunk 1]
```diff
@@ -54,100 +52,88 @@
...
-  int sendto(int fd, void *buf, int len, int flags, struct sockaddr const *to, int tolen);
-  int sendmsg(int fd, struct msghdr *m, int flags, void *pOLP = nullptr);
-  int64_t lseek(int fd, off_t offset, int whence);
-  int fstat(int fd, struct stat *);
-  int unlink(char *buf);
-  int fsync(int fildes);
-  int ftruncate(int fildes, off_t length);
-  int lockf(int fildes, int function, off_t size);
-  int poll(struct pollfd *fds, unsigned long nfds, int timeout);
+namespace SocketManager
```
Comment (brbzull0) [inline iocore/eventsystem/I_SocketManager.h:61]: not an issue.
I'd use a different name for the namespace, `net_utils` or something like that.

**Review by cmcfarlen — APPROVED** [review]
This looks good.  The only class member `pagesize` is never even used, so no need for an instance at all here.

**Comment by bryancall** [comment]
[approve ci autest]

**Review by bryancall — APPROVED** [review]
👍🏻

**Pull request merged**